code/processes/wdb.q

```diff
@@ -311,44 +324,46 @@ merge:{[dir;pt;tableinfo;mergelimits;hdbsettings;mergemethod]
+endofdaymerge:{[dir;pt;tablist;mergelimits;hdbsettings;mergemethod]
   /- merge data from partitons
-  $[(0 < count .z.pd[]) and ((system "s")<0);
-    [.lg.o[`merge;"merging on worker"];
-     {(neg x)(`.wdb.reloadsymfile;y);(neg x)(::)}[;.Q.dd[hdbsettings `hdbdir;`sym]] each .z.pd[];
-     /-upsert .merge.partsize data to sort workers, only needed for part and hybrid method 
-     if[(mergemode~`hybrid)or(mergemode~`part);
-       {(neg x)(upsert;`.merge.partsizes;y);(neg x)(::)}[;.merge.partsizes] each .z.pd[];
-       ];
-     merge[dir;pt;;mergelimits;hdbsettings;mergemethod] peach flip (key tablist;value tablist);
-     /-clear out in memory table, .merge.partsizes, and call sort worker processes to do the same
-     .lg.o[`eod;"Delete from partsizes"];
-     delete from `.merge.partsizes;
-     {(neg x)({.lg.o[`eod;"Delete from partsizes"];
-               delete from `.merge.partsizes;
-               /- run a garbage collection if enabled
-               if[gc;.gc.run[]]};`);(neg x)(::)} each .z.pd[];
-    ];	
-    [.lg.o[`merge;"merging on main"];
-     reloadsymfile[.Q.dd[hdbsettings `hdbdir;`sym]];
-     merge[dir;pt;;mergelimits;hdbsettings;mergemethod] each flip (key tablist;value tablist);
-     .lg.o[`eod;"Delete from partsizes"];
-     delete from `.merge.partsizes;
-    ]
-   ];
-  /- if path exists, delete it
-  if[not () ~ key savedir; 
-    .lg.o[`merge;"deleting temp storage directory"];
-    .os.deldir .os.pth[string[` sv savedir,`$string[pt]]];
-    ];
-  /-call the posteod function
-  .save.postreplay[hdbsettings[`hdbdir];pt];
-  $[permitreload; 
-    doreload[pt];
-    if[gc;.gc.run[]];
-    ];
-  };
-	
+  /- .z.pd funciton in finspace will cause an error. Add in this flag to skip over the use of .z.pd. This should be temporary and will be removed when issue resolved by AWS.
+  if[not .finspace.enabled;
+          $[(0 < count .z.pd[]) and ((system "s")<0);
+                  [.lg.o[`merge;"merging on worker"];
+                  {(neg x)(`.wdb.reloadsymfile;y);(neg x)(::)}[;.Q.dd[hdbsettings `hdbdir;`sym]] each .z.pd[];
+                  /-upsert .merge.partsize data to sort workers, only needed for part and hybrid method
+                  if[(mergemode~`hybrid)or(mergemode~`part);
+                          {(neg x)(upsert;`.merge.partsizes;y);(neg x)(::)}[;.merge.partsizes] each .z.pd[];
+                          ];
+                  merge[dir;pt;;mergelimits;hdbsettings;mergemethod] peach flip (key tablist;value tablist);
+                  /-clear out in memory table, .merge.partsizes, and call sort worker processes to do the same
+                  .lg.o[`eod;"Delete from partsizes"];
+                  delete from `.merge.partsizes;
+                  {(neg x)({.lg.o[`eod;"Delete from partsizes"];
+                  delete from `.merge.partsizes;
+                  /- run a garbage collection if enabled
+                  if[gc;.gc.run[]]};`);
+                          (neg x)(::)} each .z.pd[];
```

Comment:
is there a meaningful change here that I'm missing? or just formatting change?

Reply:
The only change to the eodmerge and eodsortdate functions are the addition of a  if[not .finspace.enabled; flag in order to skip over the section of the function which uses the .z.pd function. The format is changed just as a result of the extra if statement.

Reply:
> The format is changed just as a result of the extra if statement.

this specific line used to be one line and is now two, so I don't think that's completely correct